site/docs/voice/webhooks/mediaStreamRejected.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: mediaStreamRejected
+title: Media Stream Rejected
+slug: /voice/webhooks/mediaStreamRejected
+description: A general overview of Bandwidth's Media Stream Rejected Webhook
+keywords:
+  - bandwidth
+  - voice
+  - webhook
+  - streaming
+  - rejected
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This event may be sent to the url specified when sending a [`<StartStream>`][1] verb. This event will be sent if the stream requested in the `<StartStream>` verb was not started.
+
+## Request Parameters
+
+| Property                        | Description  |
+|:--------------------------------|:-------------|
+| accountId                       | The user account associated with the call |
+| answerTime                      | Time the call was answered, in ISO 8601 format |
+| applicationId                   | The id of the application associated with the call |
+| callId                          | The call id associated with the event |
+| callUrl                         | The URL of the call associated with the event |
+| cause                           | The reason the stream was rejected, can be one of `closed`, `connection-error`, `duplicate`, `error` or `track-limit` |
```

Comment:
I don't think it can be rejected with `closed`
```suggestion
| cause                           | The reason the stream was rejected, can be one of `connection-error`, `duplicate`, `error` or `track-limit` |
```